mantidimaging/core/configs/recon_config.py

```diff
@@ -123,6 +133,10 @@ def handle_special_arguments(self):
                     "Slice Indices (len {1})!".format(len_cors,
                                                       len_cor_slices))
 
+        # TODO
+        print(self.func.cors)
```

Comment:
Should these print statements be left here?

Reply:
Nope, that is a mistake.